net/geoip-shell/Makefile

```diff
@@ -0,0 +1,162 @@
+# Copyright 2024 friendly-bits, antonk (antonk.d3v@gmail.com)
+# This is free software, licensed under the GNU General Public License v3.
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=geoip-shell
+PKG_VERSION:=0.5
+PKG_RELEASE:=r2
```

Comment:
The PKG_RELEASE has the extra "r" in its release, leading to extra r also in the final filename:
`geoip-shell_0.5-rr2_all.ipk`
( https://downloads.openwrt.org/snapshots/packages/arm_cortex-a9_neon/packages/ )

The r is automatically added by the build system.

I will remove it.

Reply:
@hnyman That's odd - When building on my machine, the filename comes out correct. Should I remove the 'r' in the Makefile in future releases? So it'll be like `PKG_RELEASE:=1` ?

Reply:
> When building on my machine, the filename comes out correct. 

Are you sure that you are compiling it with really up-to-date OpenWrt master ?
The logic regarding "r" addition was added just a few weeks ago. If your build env does not  have up-to-date code, it won't have the logic.

(the whole 'r' there is quite new)

Reply:
> Are you sure that you are compiling it with really up-to-date OpenWrt master ?

Ah yes, pulled latest master and now filename has 2 r's.